### DISCUSSION TITLE
Do I need to modify the docker-compose file to add social logins?

### DISCUSSION THREAD

**Discussion by Yasas4D** [digitalocean/supabase-on-do#30] [Q&A] [answered]
Is there any easy way to add additional configurations to supabase? Also, the environmental variables that are changing from terraform would not reflect in the deployed instance. (Ex-> ENABLE_EMAIL_AUTOCONFIRM in .env.tftpl)

**Comment by timothymamo** [comment, marked as answer]
Hey @Yasas4D, if you want to add any additional configuration to supabase you can do this by:
1. adding the variable to the appropriate service in the [docker-compose.yml](https://github.com/digitalocean/supabase-on-do/blob/main/packer/supabase/docker-compose.yml)
2. adding the variable to the the env.tftpl (using your example I would add `ENABLE_EMAIL_AUTOCONFIRM=${TF_ENABLE_EMAIL_AUTOCONFIRM}`)
3. Add the TF variable in the locals [`env_file`]( https://github.com/digitalocean/supabase-on-do/blob/main/terraform/locals.tf#L112) section (e.g. `TF_ENABLE_EMAIL_AUTOCONFIRM = "${var.enable_email_autoconfirm}"` or set as `TF_ENABLE_EMAIL_AUTOCONFIRM = true` if you don't need to change this)
4. Optional (according to how you set the varible in step 3): Add the variable in the [varibales.tf](https://github.com/digitalocean/supabase-on-do/blob/main/terraform/variables.tf) file (e.g.
```
variable "enable_email_autoconfirm" {
  description = "Boolean enabling email autoconfirmation."
  type        = bool
  default = true
}
```
Hope this helps

**Reply by Yasas4D** [reply]
Thank you very much @timothymamo
